protocol/communities/community.go

```diff
@@ -52,7 +52,7 @@ type Config struct {
 	Logger                              *zap.Logger
 	RequestedToJoinAt                   uint64
 	RequestsToJoin                      []*RequestToJoin
-	MemberIdentity                      *ecdsa.PublicKey
+	MemberIdentity                      *ecdsa.PrivateKey
```

Comment:
Why do you need it to be the private key instead?

Reply:
We need a private key to create a shared secret with the owner.

https://github.com/status-im/status-go/blob/becf2b8e4b93f2a076d81d49648ebb991e5f5b76/protocol/communities/community.go#L916

then :point_down: 

https://github.com/status-im/status-go/blob/becf2b8e4b93f2a076d81d49648ebb991e5f5b76/protocol/communities/community_bloom_filter.go#L94